pkg/signature/kms/cliplugin/common/interface_test.go

```diff
@@ -72,6 +72,26 @@ func TestPluginArgsJSON(t *testing.T) {
 	},
 	"methodName": "defaultAlgorithm",
 	"defaultAlgorithm": {}
+}`,
+		},
+		{
+			name: "supportedAlgorithms",
+			pluginArgs: &PluginArgs{
+				InitOptions: testInitOptions,
+				MethodArgs: &MethodArgs{
+					MethodName:          SupportedAlgorithmsMethodName,
+					SupportedAlgorithms: &SupportedAlgorithmsArgs{},
+				},
+			},
+			want: `{
+	"initOptions": {
```

Comment:
nit: this `initOptions` looks repeated everywhere in this test, but I don't know if it matters. Perhaps we could do some deduplication. But really up to you.

Reply:
I think we should keep it, just so folks can easily see complete examples of PluginArgs.